src/content/en/updates/posts/2016/04/chrome-51-deprecations.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: updates/post
+title: "API Deprecations and Removals in Chrome 51"
+description: "An round up of the deprecations and removals in Chrome to help you plan."
+published_on: 2016-04-20
+updated_on: 2016-04-20
+authors:
+  - josephmedley
+tags:
+  - deprecations
+  - removals
+  - chrome51
+featured_image: /web/updates/images/2016/04/chrome-51-deprecations/deps-removes.png
```

Comment:
The image path is wrong. It should be

/web/updates/images/2016/04/chrome-51-deprecations/deps-rems.png